setup.py

```diff
@@ -8,6 +8,8 @@
 import platform
 import sys
 
+import pkg_resources
```

Comment:
This is a slow import on systems with lots of packages IIRC, so let's lazy import it in the failure path.

Reply:
Good point! I have moved the import.